README.md

```diff
@@ -59,7 +59,10 @@ const App = () => {
     <SortableList onSortEnd={onSortEnd} className="list" draggedItemClassName="dragged">
       {items.map((item) => (
         <SortableItem key={item}>
-          <div className="item">{item}</div>
+          <div className="item">
```

Comment:
I actually would not show the `SortableKnob` as part of the `basic example`. Let me add another example in the `SortableKnow` section.